.travis.yml

```diff
@@ -12,10 +12,14 @@ env:
   - ENGINE=GEOS
 
 before_script:
+  - if [[ $ENGINE = PDO_PGSQL ]]; then psql -c 'create database geo_tests;' -U postgres; fi;
+  - if [[ $ENGINE = PDO_PGSQL ]]; then psql -c 'create database geo_tests_tmp;' -U postgres; fi;
   - if [[ $ENGINE = PDO_MYSQL ]]; then bash .travis.install-mysql.sh; fi;
+  - if [[ $ENGINE = PDO_MYSQL ]]; then mysql -e 'create database geo_tests;'; fi;
+  - if [[ $ENGINE = PDO_MYSQL ]]; then mysql -e 'create database geo_tests_tmp;'; fi;
   - if [[ $ENGINE = GEOS ]]; then bash .travis.install-geos.sh; fi;
   - if [[ $ENGINE = SQLite3 ]]; then bash .travis.install-sqlite3.sh; fi;
-  - composer install
+  - composer install --prefer-source
```

Comment:
The doc says that these settings only apply to *projects*, so it should only apply when you run composer install from within the brick\geo repository (that is, when you're developing the library) not when it's included as a dependency of a third-party project!